content: Effortless Solutions success story and blog post (#406)
boxyhq/website: blog/2024-03-26-how-effortless-solutions-managed-multi-tiered-client-needs-with-boxyhq-sso.md

---
description: 'Discover how Effortless Solutions expertly navigated multi-tiered client needs using BoxyHQ SSO, achieving seamless integration and enhanced security.'
slug: how-effortless-solutions-managed-multi-tiered-client-needs-with-boxyhq-sso
title: 'How Effortless Solutions Managed Multi-Tiered Client Needs with BoxyHQ SSO'
tags_disabled: [security, sso, enterpriseready, bubble, integration]
image: /images/success-stories/effortless-solutions-case-study.png
hide_table_of_contents: true
author: Schalk Neethling
author_title: Customer Success, Community, DevRel - @BoxyHQ
author_url: https://github.com/schalkneethling
author_image_url: /images/authors/schalk.jpg
---

import FeaturedQuote from "../src/components/FeaturedQuote";
import LearnMoreCTA from "../src/components/ctas/LearnMoreCTA";
import SignupCTA from "../src/components/ctas/SignupCTA";

> Efficiency and security are critical when implementing high-impact solutions leaving Effortless Solutions with a unique challenge: implementing an Enterprise Single Sign-On (SSO) solution to meet the needs of a client's client in the Netherlands. The requirement was for a SaaS application developed on the Bubble platform to integrate seamlessly with Microsoft EntraID, all while ensuring little to no disruption and downtime. This emphasized the need for a streamlined implementation process that would enhance productivity while maintaining high-security standards.
> 
> The search for a suitable SSO solution led Effortless Solutions to select BoxyHQ. This decision was driven by several factors: positive customer feedback, the open-source nature of BoxyHQ which mitigated vendor lock-in concerns, and seamless integration with Bubble.

Author: ```suggestion
Effortless Solutions faced a unique challenge: implementing an Enterprise Single Sign-On (SSO) solution for a customer's client in the Netherlands, requiring a SaaS application developed on the Bubble platform to seamlessly integrate with Microsoft EntraID, ensuring minimal disruption and downtime while enhancing productivity and maintaining high-security standards. After careful consideration, Effortless Solutions selected BoxyHQ due to positive customer feedback, its open-source nature mitigating vendor lock-in concerns, and seamless integration with Bubble.
```

<LearnMoreCTA label="Learn more about Effortless Solutions" url="https://effortlesssolutions.fr/" />

## What was the motivation to implement Enterprise SSO?

> **Ahmed Elkaffas (Effortless Solutions):** The motivation to implement [enterprise Single Sign-On (SSO)](/enterprise-sso) came from a specific requirement of one of our customer's clients in the Netherlands. This client, using the software as a service (SaaS) application developed on the Bubble platform, wanted their application to seamlessly integrate with their existing Microsoft EntraID infrastructure.

Author: ```suggestion
**Ahmed Elkaffas (Effortless Solutions):** The motivation to implement [enterprise Single Sign-On (SSO)](/enterprise-sso) came from a specific requirement of one of our customer's clients in the Netherlands. This client, using a software as a service (SaaS) application developed on the Bubble platform, wanted their application to seamlessly integrate with their existing Microsoft EntraID infrastructure.
```

> **Ahmed Elkaffas (Effortless Solutions):** The motivation to implement [enterprise Single Sign-On (SSO)](/enterprise-sso) for Effortless Solutions came from a specific requirement of one of their customer's clients in the Netherlands. This client, using the software as a service (SaaS) application developed on the Bubble platform, wanted their application to seamlessly integrate with their existing Microsoft EntraID infrastructure.

Author: ```suggestion
**Ahmed Elkaffas (Effortless Solutions):** The motivation to implement [enterprise Single Sign-On (SSO)](/enterprise-sso) for Effortless Solutions came from a specific requirement of one of our customer's clients in the Netherlands. This client, using the software as a service (SaaS) application developed on the Bubble platform, wanted their application to seamlessly integrate with their existing Microsoft EntraID infrastructure.
```

> They aimed to create a seamless, efficient login process for employees, eliminating the need for multiple logins and thereby increasing productivity and ensuring a higher level of security compliance. This need for a streamlined authentication process that could integrate well with the customer's existing systems and protocols was the main driving force behind seeking an enterprise SSO solution, leading to the selection and implementation of BoxyHQ.

Author: ```suggestion
They aimed to create a seamless, efficient login process for employees, eliminating the need for multiple logins and thereby increasing productivity and ensuring a higher level of security compliance. This need for a streamlined authentication process that could integrate well with the customer's existing systems and protocols was the main driving force behind seeking an enterprise SSO solution, leading to the selection and implementation of BoxyHQ SaaS.
```

- **Seamless Integration with Bubble**: BoxyHQ offered a seamless integration with Bubble, which was essential for us. We sought an SSO solution that could be integrated without the hassle of developing or adopting new plugins, as this ran the risk of disrupting operations or delaying deployment.
- **Rapid Deployment and Comprehensive Security Compliance**: The ease of adoption of BoxyHQ, combined with its compliance with stringent security standards, was crucial, particularly in the European context where security compliance is not optional but mandatory.

These factors collectively contributed to the decision to implement BoxyHQ as our chosen enterprise SSO solution, addressing the specific needs of our customer and their end-users while aligning with their broader security, flexibility, and integration requirements.

<SignupCTA campaign="blog-effortless-solutions" />

## Can you elaborate on the direct benefits and second-order benefits you observed following the adoption of BoxyHQ?

> **Ahmed (Effortless Solutions):** Following the adoption of BoxyHQ, we observed several direct and second-order benefits that significantly impact our operations, customer satisfaction, and the overall value delivered to our clients:

Author: ```suggestion
**Ahmed (Effortless Solutions):** Following the adoption of BoxyHQ, we observed several direct and second-order benefits that significantly impacted our operations, customer satisfaction, and the overall value delivered to our clients:
```

- **Operational Efficiencies**: The ease and speed of implementing BoxyHQ allowed us to allocate resources more efficiently, avoiding the extended timelines and resource commitments often associated with deploying new IT solutions.
- **Reputation and Market Positioning**: Successfully implementing an enterprise-grade SSO solution with BoxyHQ enhanced our reputation as a capable no-code agency that can navigate and deliver complex IT solutions. This success positions us favorably in the market, potentially attracting more clients seeking similar solutions.

By addressing the specific needs of our client, we not only met the immediate technical and security requirements but also realized broader benefits that strengthened our market position, enhanced operational capabilities, and allowed us to build even stronger relationships with our clients.

## In terms of business impact, how much time did it save Effortless Solutions?

> **Ahmed (Effortless Solutions):** Effortless Solutions experienced significant time savings by choosing BoxyHQ for their enterprise SSO implementation. The expectation that the solution could be deployed within a matter of days was a key advantage over other potential solutions that might have taken months to implement.

Author: ```suggestion
**Ahmed (Effortless Solutions):** We experienced significant time savings by choosing BoxyHQ for our enterprise SSO implementation. The expectation that the solution could be deployed within a matter of days was a key advantage over other potential solutions that might have taken months to implement. This was then also our experience during implementation. Not only did the production deployment go smoothly, but it was completed in days with no disruption to any of the client's end users.
```

> **Ahmed (Effortless Solutions):** Effortless Solutions experienced significant time savings by choosing BoxyHQ for their enterprise SSO implementation. While the document doesn't specify the exact number of hours or days saved, Ahmed Elkaffas, the founder, emphasized the rapid deployment capability of BoxyHQ as a critical factor in their decision-making process. He noted the expectation that the solution could be deployed within a matter of days, which was a key advantage over other potential solutions that might have taken months to implement.

Author: ```suggestion
**Ahmed (Effortless Solutions):** Effortless Solutions experienced significant time savings by choosing BoxyHQ for their enterprise SSO implementation. The expectation that the solution could be deployed within a matter of days was a key advantage over other potential solutions that might have taken months to implement.
```

<FeaturedQuote personName="Ahmed Elkaffas" personRole="Founder - Effortless Solutions" pictureSrc="/images/success-stories/ahmed-effortless-solutions">
 We encountered a problem during testing on the development environment which we reported to BoxyHQ on Friday afternoon. On Monday, BoxyHQ had fixed and deployed a new version ready for us to test. We tested it, and it worked fantastically.
</FeaturedQuote>

## Have you seen any significant growth in acquiring enterprise clients as a direct result of utilizing SSO?

**Ahmed (Effortless Solutions):** Now that we have successfully deployed BoxyHQ for one of our clients, we can approach future clients with confidence knowing that we can rely on the support of BoxyHQ and that we can guarantee an implementation timeline of days instead of weeks or months. This changes everything for us and opens new doors for Effortless Solutions.

> Because of other BoxyHQ SaaS features such as Directory Sync, we can offer services to ever larger clients laying the foundation for the future growth and continued success of our company. We wish to thank everyone at BoxyHQ for their dedication to the success of our implementation and for being a partner in our success.

Author: ```suggestion
Because of other BoxyHQ SaaS features such as [Directory Sync](/directory-sync), we can offer services to ever larger clients laying the foundation for the future growth and continued success of our company. We wish to thank everyone at BoxyHQ for their dedication to the success of our implementation and for being a partner in our success.
```
